tests/unit/jobserver/test_utils.py

```diff
@@ -9,6 +9,14 @@ def test_dotted_path():
     assert dotted_path(OutputChecker) == "jobserver.authorization.roles.OutputChecker"
 
 
+def test_is_safe_path_with_safe_path():
+    assert is_safe_path("/")
```

Comment:
Could you also check a path with a component/segment/some text in it alongside this check? (experience with path handling and single slashes has caused me issues in the past!)

Reply:
I've added `assert is_safe_path("/status/")` to the same test.